seqr/management/commands/export_cpg_id_map.py

```diff
@@ -0,0 +1,57 @@
+from django.core.management.base import BaseCommand
+
+import csv
+from seqr.models import Sample
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = (
+        "Export a tsv file with the cpg_id to individual/family/project_guid mappings"
+    )
+
+    def handle(self, *args, **options):
+        outfile_name = '/app/seqr/cpg_id_map.tsv'
+
+        # Get all the active samples
+        samples = Sample.objects.filter(is_active=True)
+
+        logger.info(f"Exporting {len(samples)} active samples to a tsv file")
+
+        sample_id_to_project_guid = {}
+        sample_id_to_family_guid = {}
+        sample_id_to_individual_id = {}
+
+        for sample in samples:
+            sample_id = sample.sample_id
+            individual_id = sample.individual.individual_id
+            family_guid = sample.individual.family.guid
+            project_guid = sample.individual.family.project.guid
+
+            if sample_id not in sample_id_to_project_guid:
+                sample_id_to_project_guid[sample_id] = project_guid
+            if sample_id not in sample_id_to_family_guid:
+                sample_id_to_family_guid[sample_id] = family_guid
+            if sample_id not in sample_id_to_individual_id:
+                sample_id_to_individual_id[sample_id] = individual_id
+
+        # Write the sample_id_to_individual/family/project_guid dicts to a tsv file
+        with open(outfile_name, "w") as f:
+            writer = csv.writer(f, delimiter="\t")
+            writer.writerow(["cpg_id", "individual_id", "family_guid", "project_guid"])
+            for sample_id, project_guid in sample_id_to_project_guid.items():
+                writer.writerow(
+                    [
+                        sample_id,
+                        sample_id_to_individual_id[sample_id],
+                        sample_id_to_family_guid[sample_id],
+                        project_guid,
+                    ]
+                )
```

Comment:
You have an ORM, let it do the work for you. Try something like this:

```suggestion
        # Write the sample_id_to_individual/family/project_guid dicts to a tsv file
        with open(outfile_name, "w") as f:
            writer = csv.writer(f, delimiter="\t")
            writer.writerow(["cpg_id", "individual_id", "family_guid", "project_guid"])
       
       for s in Sample.objects.filter(is_active=True).select_related('individual__family__project'):
            writer.writerow(
                    [
                        s.sample_id,
                        s.individual.individual_id,
                        s.individual.family.guid,
                        s.individual.family.project.guid,
                    ]               


```